tools/link.py

```diff
@@ -2390,35 +2390,30 @@ def modularize():
     diagnostics.warning('emcc', 'EXPORT_NAME should not be named "config" when targeting Safari')
 
   if settings.MODULARIZE == 'instance':
-    src = '''
+    wrapper_funcion = '''
```

Comment:
```suggestion
    wrapper_function = '''
```
also, this could be a f-string `f''' .. {generated_js}`

Reply:
I think it was deliberate in this case since it would force all the actual curly braces to be escaped (with `{{` and `}}`)